README.md

```diff
@@ -169,3 +169,52 @@ The Short-Night engine. see:
 
 - https://foxzilla.github.io/short-night/globals.html
 - https://github.com/FoXZilla/short-night/blob/master/src/index.ts
+
+# Contributor's Guide
```

Comment:
The steps you wrote is just basic knowledge on Github. We should move focus on how to run this Repo on your PC. E.g. Run `npm run dev` then open `docs/webpack.html`